### PR TITLE
[modify] css styling and item details view

### PR DESCRIPTION
**Pull request by josephlin55555**

### CODE REVIEW

**Comment by vtumrukota** [comment]
Looks good - Only thing is lets be more descriptive with our CSS class variable names moving forward

File: snapmarketapp/app/scripts/app.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ angular.module('starter', ['ionic', 'starter.controllers', 'starter.services', '
     }
   })
     .state('tab.buyItemDetail', {
```
Comment (andyttran) [inline snapmarketapp/app/scripts/app.js]: change tab.buyItemDetail to tab.buyListingDetail

File: snapmarketapp/app/scripts/app.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ angular.module('starter', ['ionic', 'starter.controllers', 'starter.services', '
...
-      url: '/buySearch/:itemId',
+      url: '/buySearch/:listingId',
       views: {
         'buy': {
           templateUrl: 'templates/buy-itemDetail.html',
```
Comment (andyttran) [inline snapmarketapp/app/scripts/app.js]: see prev comment

File: snapmarketapp/app/scripts/buy/buy-controllers.js [hunk 3]
```diff
@@ -33,7 +33,9 @@ angular.module('buy.controllers', ['firebase'])
   $scope.selectCard = function(card) {
     //card argument is listing object
     $rootScope.currentListing = card;
-    $state.go('tab.buyItemDetail');
+
+    //$id is the unique hash id for each item in the firebase array
+    $state.go('tab.buyItemDetail', {'listingId': card.$id});
```
Comment (andyttran) [inline snapmarketapp/app/scripts/buy/buy-controllers.js]: see prev comment

**Pull request merged**